doc/source/metadata/Probabilities.rst

```diff
@@ -0,0 +1,179 @@
+.. _prob-section:
+
+Probability Distribution
+========================
+
+.. contents:: Contents
+    :depth: 3
+
+Context
+-------
+
+As IMPROVER is inherently probabilistic,
+it seems appropriate to have a section specifically focused
+on the representation of probability distributions in the metadata.
+This included some extensions to the `CF Metadata Conventions`_ 
+which provide limited support in this area.
+
+Probabality distributions can be represented in one of three different ways: 
```

Comment:
```suggestion
Probability distributions can be represented in one of three different ways: 
```

Reply:
I really can't spell that word! - corrected.